src/core/pool_type.hpp

```diff
@@ -83,6 +84,9 @@ struct Pool : PoolBase {
 
 	static constexpr size_t MAX_SIZE = Tmax_size; ///< Make template parameter accessible from outside
 
+	using BitmapStorage = uint64_t;
+	static constexpr size_t BITMAP_SIZE = std::numeric_limits<BitmapStorage>::digits;
```

Comment:
Maybe change this to a plain `uint`, in part to silence the warning produced for MSVC builds, and in part just because it isn't actually used an index into an STL container.